Add minimum amount for Visa captures and refunds (#206)
clearhaus/gateway-api-docs: website/content/gateway/api_reference/changes/changes.md

Sorted by descending timestamp.

> #### Add minimum amount for Visa refunds and captures

Reviewer: ```suggestion
#### Add minimum amount for Visa captures and refunds
```

https://github.com/clearhaus/gateway-api-docs/pull/206/files#r2107712678

> Starting 2025-XX-XX, we will enforce a minimum amount of USD 0.005 for refunds

Author: This is the effective date which it why it's in the past

> and captures made with Visa. This is due to USD 0.005 being half a minor and
> anything less be rounded to 0.

Reviewer: ```suggestion
and captures made with Visa. This is due to USD 0.005 being half a minor, and
a smaller value will be rounded to 0.
```

> a smaller value will be rounded to 0.

Author: ```suggestion
a smaller value will be rounded to 0 by Visa.
```
